app/lib/signup/domains_builder.rb

```diff
@@ -15,7 +15,7 @@ def generate
     end
 
     def generate_subdomain
-      result = org_name.to_s.parameterize
+      result = org_name.tr("_", "-").to_s.parameterize
```

Comment:
```suggestion
      result = org_name.to_s.tr('_', '-').parameterize
```

Reply:
Let me check this on my  local then I will commit this suggestion